app/views/schools/advice/total_energy_use/_insights.html.erb

```diff
@@ -11,6 +11,7 @@
     <%= @overview_data.date_ranges %>
     <%= link_to_help_for_feature :management_summary_overview, title: "#{t('schools.show.more_information')} #{fa_icon('info-circle')}".html_safe %>
   </div>
+  <%= render 'schools/advice/how_have_we_analysed_your_data_table_caption', data_target: "how-have-we-analysed-your-data-footnotes" %>
```

Comment:
This doesn't look right on the page, as there's already a table caption in the previous div. Suggest you remove the old one from here and just leave the new one instead.